android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -80,10 +81,13 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
       is ReactEditText -> this.hook = EditTextHook(this, view)
       is ReactSwipeRefreshLayout -> this.hook = SwipeRefreshLayoutHook(this, view)
       is ReactScrollView -> this.hook = ScrollViewHook()
+      is ReactHorizontalScrollView -> this.hook = ScrollViewHook()
       is ReactViewGroup -> this.hook = ReactViewGroupHook()
     }
   }
 
+  override fun isContinuous() = view is ReactScrollView || view is ReactHorizontalScrollView || view is ReactSwipeRefreshLayout
```

Comment:
Move this to the hook, please.

It also may be worth it to consider using ternary logic here (`true`, `false`, `null`) and return null for the views that use the default hook to not make any assumptions.

Reply:
Moved isContinuous to the hook in [475c37d](https://github.com/software-mansion/react-native-gesture-handler/pull/3264/commits/475c37deeda4cef01af0a9a679c26526fa24ce1e).

Regarding the ternary logic, I'm not sure if we can apply it to the `NativeViewGestureHandler`. By default, `isContinuous` has to be set to `false`, while `NativeViewGestureHandlerHook` is both the the default hook, and the common ancestor of all other hooks. 

On the other hand, I think it makes sense to use the ternary logic on the base `GestureHandler` class, as it's not directly used anywhere, and it'd be preferable for every old and new gesture handler to explicitly set the `isContinuous` function to either `true` or `false`.

Reply:
Yeah, that's what I meant:
- change `isContinuous` to return `Boolean?`
- make every gesture handler explicitly implement it to return true or false
- make basic hook return null
- update other hooks to either return `true` or `false`

Reply:
I think it'd make more sense to convert `GestureHandler` to be an abstract class. I think that just making `isContinuous` a ternary isn't enough to signal that `GestureHandler`'s children should implement it; same for most of the other GestureHandle functions which behave like `abstract` functions - are left empty, and expected to be implemented, while not being explicitly marked as `abstract`.

I don't think `GestureHandler` class is used anywhere directly, and there are multiple such functions which it does not implement, while expecting it's children to implement them.

Of course if that's a change that you think makes sense to implement, i'll open a separate PR for it.